mindsdb/api/mysql/mysql_proxy/utilities/lightwood_dtype.py

```diff
@@ -0,0 +1,48 @@
+class dtype:
```

Comment:
See the above comments, I think this shouldn't be used. Else, we are forcing Lightwood philosophy on other handlers that may come after. 

The solution may require additional discussions, but I think it is something like guaranteeing predictive handlers will return data with the "correct"  data types depending on what data handler called them.